cloud_experts_tutorials/cloud-experts-deploy-api-data-protection.adoc

```diff
@@ -279,7 +279,7 @@ metadata:
  name: ${CLUSTER_NAME}-dpa
  namespace: openshift-adp
 spec:
- backupImages: false
+ backupImages:
```

Comment:
Instead of keeping this value to empty, lets set it as true

Reply:
Done, thanks!